examples/src/lib.rs

```diff
@@ -73,18 +69,11 @@ impl ExampleOptions {
             3 => FieldExtension::Cubic,
             val => panic!("'{}' is not a valid field extension option", val),
         };
-        let hash_fn = match self.hash_fn.as_str() {
-            "blake3_192" => HashFunction::Blake3_192,
-            "blake3_256" => HashFunction::Blake3_256,
-            "sha3_256" => HashFunction::Sha3_256,
-            val => panic!("'{}' is not a valid hash function option", val),
-        };
```

Comment:
We shouldn't get rid of these. Instead, we should figure out how we can update examples to support instantiations over multiple hash functions.

While we are trying to figure out how to do this, it might be easier to turn off all examples except for one (e.g., `fib2`). Once we have the `fib2` example working with multiple hash functions, we can re-enable all other examples.

Reply:
I've commented out all the other examples in relevant places and refactored the `fib2` example to support instantiations over multiple hash functions. As per our discussion, I've declared both `FibProver` and `FibExample` generically to accept type `ElementHasher`. Please see [62ca1c8](https://github.com/novifinancial/winterfell/pull/111/commits/62ca1c881c0193401878fbf5f2175a7e8ac3eda9).